app/views/statuses/edit.html.erb

```diff
@@ -27,19 +27,20 @@ See COPYRIGHT and LICENSE files for more details.
 
 ++#%>
 
-<% html_title t(:label_administration), "#{t(:label_edit)} #{Status.model_name.human} #{h @status.name}" %>
+<% html_title t(:label_administration), "#{t(:label_edit)} #{Status.model_name.human} #{h @status.name_was}" %>
 
 <%=
   render Primer::OpenProject::PageHeader.new do |header|
-    header.with_title { @status.name }
+    header.with_title { @status.name_was }
     header.with_breadcrumbs([{ href: admin_index_path, text: t("label_administration") },
                              { href: admin_settings_work_packages_general_path, text: t(:label_work_package_plural) },
                              { href: statuses_path, text: t(:label_status) },
-                             @status.name])
+                             @status.name_was])
   end
 %>
 
-<%= labelled_tabular_form_for(@status) do |f| %>
-  <%= render partial: 'form', locals: { f: f } %>
-  <%= styled_button_tag t(:button_save), class: '-primary -with-icon icon-checkmark' %>
+<%= error_messages_for :status %>
```

Comment:
💡 I don't know if there are error cases defined for status on `base` rather than on an attribute. The way the errors are currently displayed is fine but duplicates the message:

<img width="765" alt="image" src="https://github.com/user-attachments/assets/af68bd01-d81b-4c18-af36-2c0503d1a356">

Out of scope for this PR but changing the error message helper so that e.g. via a flag it only displays errors on `base` could be a simple improvement.

Reply:
Yes, but showing `base` errors only could be problematic too: checkboxes do not have any inline validation message, unless they are part of a checkbox group, and this checkbox group must have a name matching the one in the model to get the inline validation message being displayed.

To try it, check both "is default" and "is readonly". There will be an error message on "is default" saying that it can't be active when "is readonly" is also checked. It is displayed in the error banner, but not inline. That's a bit of a regression compared to how the page behaved as before the label and the box border would be in red (without message though).

Anyway, this has to be solved globally, and I think Henriette is the person who should coordinate that. I'll show her the page to get her feedback about it.